md/access-control-api.md

```diff
@@ -0,0 +1,61 @@
+# Access control API
+
+::: info
+**Note:** This API is only available in Efficiency and Performance edition.
+:::
+
+## BDM
+
+#### Description
+
+Access control is available to protect the tenant BDM. You can use this API to get the access control status (lastUpdatedBy, lastUpdateDate...).
+
+#### Methods
+ * GET - Read access control status.
+ * DELETE - Remove access control
+ 
+::: info
+**Note:** A servlet to import access control file exists. You can use it with this url `services/bdmAccessControl/install?bdmAccessControlUpload=accessControlFileToUpload.xml`. 
+  The upload needs to be performed before (POST portal/bdmAccessControlUpload).
+:::
+
+
+
+#### Get access Control status
+
+Make this call to get the access control status.
+
+* **URL**  
+  `/API/accessControl/bdm`  
+* **Method**  
+  `Get`
+* **Success Response**
+    * Code: 200
+    * Payload:      
+  ```json
+  {
+    "id": "4090",
+    "name": "file_name.xml",
+    "type": "BDM_ACCESS_CONTROL",
+    "state": "INSTALLED",
+    "lastUpdatedBy": "4",
+    "lastUpdateDate": "2018-01-17T17:05:36.671Z"
+  }
+  ``` 
+  ::: info
+  **Note:** If `"lastUpdatedBy": "-1"` it mean that is tenant_technical_user who has installed or updated access control.
```

Comment:
mean**s** that the access control were last installed or updated by the tenant technical user